frontend/src/features/public-form/PublicFormProvider.tsx

```diff
@@ -1388,6 +1388,18 @@ export const PublicFormProvider = ({
     return <NotFoundErrorPage />
   }
 
+  const DATE_TIME_FORMAT_STRING = 'do MMM yyyy, h:mm:ss a'
+  const MOCK_CONSTANT_LAST_SAVED_DATETIME_STRING_FOR_STORYBOOK_TO_HAVE_NO_DIFFS =
+    format(new Date('2025-11-14T12:00:00.000Z'), DATE_TIME_FORMAT_STRING)
+  const actualLastSavedDateTimeString = draftSubmission?.lastUpdated
+    ? format(new Date(draftSubmission.lastUpdated), DATE_TIME_FORMAT_STRING)
+    : undefined
+  // RATIONALE: Replace with constant string if actual last saved date time string exists during storybook visual tests to prevent noisy diffs.
+  const draftLastSavedDateTimeString =
+    actualLastSavedDateTimeString && isTest
+      ? MOCK_CONSTANT_LAST_SAVED_DATETIME_STRING_FOR_STORYBOOK_TO_HAVE_NO_DIFFS
+      : actualLastSavedDateTimeString
```

Comment:
Would this also make sense? 

```suggestion
  // move these two to top-level constants
  const DATE_TIME_FORMAT_STRING = 'do MMM yyyy, h:mm:ss a'
  const MOCK_CONSTANT_LAST_SAVED_DATETIME = '2025-11-14T12:00:00.000Z'

  const lastSavedDateTime = draftSubmission?.lastUpdated && isTest
    ? MOCK_CONSTANT_LAST_SAVED_DATETIME
    : draftSubmission?.lastUpdated
    
  const lastSavedDateTimeString = lastSavedDateTime
    ? format(new Date(lastSavedDateTime), DATE_TIME_FORMAT_STRING)
    : undefined
```

Reply:
Yeah - this is more concise, will apply the fix